### PR TITLE
TFA Fix for Scheduled scrub validation

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Fail Log- http://magna002.ceph.redhat.com/cephci-jenkins/test-runs/baremetal/RH/7.1/rhel-9/18.2.1-155/rados/30/tier-2_rados_test-scrubbing/

Individual test pass log - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UB27IR

Full suite log: 	http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2UZGV3

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
Fixed the Unsetting scrubbing issue and the pass log  is -
	http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-M5QNDL

File: tests/rados/scheduled_scrub_scenarios.py [hunk 1]
```diff
@@ -52,11 +27,18 @@ def run(ceph_cluster, **kw):
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_obj = RadosScrubber(node=cephadm)
     mon_obj = MonConfigMethods(rados_obj=rados_obj)
-    # Storing the pg dump log before setting the scrub parameters
-    before_scrub_log = rados_obj.get_pg_dump("pgid", "last_scrub_stamp")
-    before_deep_scrub_log = rados_obj.get_pg_dump("pgid", "last_deep_scrub_stamp")
-    # Preparation of cofiguration parameter values from the current
-    # cluster time
+    # Creating ec pool
+    ec_config = config.get("replicated_pool")
```
Comment (pdhiran) [inline tests/rados/scheduled_scrub_scenarios.py:31]: Looks like we are passing replicated pool configs here. Can we rename the param?

Or are we intending to pass EC config itself?

**Comment by pdhiran** [comment]
Code looks good, but I see in the logs that a few tests are consuming 3+ hours. Is it expected? Can we work on reducing the run time?

**Review by pdhiran — APPROVED** [review]
Looks good.

**Pull request merged**